### DISCUSSION TITLE
Impossible to reach the esp32cam

### DISCUSSION THREAD

**Discussion by komax74** [jomjol/AI-on-the-edge-device#1166] [Q&A] [answered]
Hi to everyone,
yesterday I did the installation for the first time and all worked perfectly, then upgraded to 12.0.1 all ok, after I power off the esp32cam for whale day.
Today, came back reattached the usb cable (the led in usb module is always red) but nothing was not responding to the ip address set yesterday.
Then not knowing how to solve I restart the whole procedure from scratch, flashed the firmware with the "initial_esp32_setup", formatted the microsd, copied all files and rebooted. Nothing still not responding, I tried to change the ip as well but nothing, always "unable to connect to the server" or "An error occurred during the connection", with a wifi scanner I checked and esp32 is active on that ip but not responding I think the webserver.
What could have happened? How can I get it running again?

**Comment by caco3** [comment]
first thing always is to check the log over USB, see https://github.com/jomjol/AI-on-the-edge-device/issues/1161#issuecomment-1277831096

**Comment by komax74** [comment]
Hi, I followed the instructions and the log says : 
E (12509) main: Failed to initialize camera module. Check that your camera module is working and connected properly.
Failed to initialize camera module. Check that your camera module is working and connected properly.

I tried to remove and attach again the cable but nothing, always the same error.
How I can fix this problem?

**Reply by caco3** [reply]
then the cable or the cam are defective or it is an unsupported cam!
do you have a photo or another module/cam to test?

**Reply by komax74** [reply]
eps32 and cam worked perfectly the first day.
However I bought two, I try to redo everything with the other module maybe this it broke.
Before it stopped working I had attached esp32cam via the usb module to a powerbank but I don't think it could have been that.

**Reply by caco3** [reply]
I don't think its the power bank, else it would not try to start anymore at all.

Depending of where you get the cam from, it could be of bad quality. I had such issues before with stuff from China, but not with the ones I use for this project.

**Reply by komax74** [reply]
I have tried with the other module and everything works, so at this point it must be a hardware defect or something is wasted, I have read that sometimes a power surge and the camera module no longer initializes.

**Comment by caco3** [comment, marked as answer]
HW issue, see https://github.com/jomjol/AI-on-the-edge-device/discussions/1166#discussioncomment-3890262
